cytoscape-panzoom.js

```diff
@@ -328,6 +347,10 @@ SOFTWARE.
             return false; // so we don't pan close to the slider handle
           });
 
+          $slider.bind("touchstart", function() {
+            return false;
```

Comment:
Hmmm... so we can never touch the slider?

Reply:
I am actually not entirely sure what this is for, I put it for the touch event because I wanted to make sure the behaviour is consistent with mouse events.

Reply:
Ah ok, fair enough; yeah if mouse events aren't affecting the slider then neither should touch events.